docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,2498 @@
+* Adds warnings for managed system policies {kibana-pull}132269[#132269]
+* Skip empty prompt screen {kibana-pull}130862[#130862]
+* Console now supports properly handling multiple requests. For es errors such as `400`, `405` exception results are displayed with successful request results in the order they called {kibana-pull}129443[#129443]
+* Display vector tile API response in Console {kibana-pull}128922[#128922]
+* Adds option to disable keyboard shortcuts {kibana-pull}128887[#128887]
+
+Maps::
+* Show marker size in legend {kibana-pull}132549[#132549]
+* Fixes marker size scale issue for counts {kibana-pull}132057[#132057]
+* Scale marker size by area {kibana-pull}131911[#131911]
+* Localized basemaps {kibana-pull}130930[#130930]
+* Support term joins for Elasticsearch document source with vector tile scaling {kibana-pull}129771[#129771]
+* Allow feature editing for document layers with "applyGlobalTime", "applyGlobalQuery", and joins {kibana-pull}124803[#124803]
+
+Observability::
+* Bumps synthetics integration package to 0.9.4 {kibana-pull}133423[#133423]
+* Immediately re-run monitors in the synthetics service when they're edited {kibana-pull}132639[#132639]
+* Enables log flyouts on APM logs tables {kibana-pull}132617[#132617]
+* Adds logging to Metric Threshold Rule {kibana-pull}132343[#132343]
+* Adds Page load distribution chart to overview page {kibana-pull}132258[#132258]
+* Show experimental locations only when a particular flag is enabled {kibana-pull}132063[#132063]
+* Trace explorer {kibana-pull}131897[#131897]
+* Static Java agent version list becomes stale quickly {kibana-pull}131759[#131759]
+* Adds recovery context to Log Threshold rule {kibana-pull}131279[#131279]
+* Adds recovery context to the Metric Threshold rule {kibana-pull}131264[#131264]
+* Adds context variables to recovery alerts for Inventory Threshold rule {kibana-pull}131199[#131199]
+* Copy alert state to alert context and implement alert recovery {kibana-pull}128693[#128693]
+* Progressive fetching (experimental) {kibana-pull}127598[#127598]
+* Replace environment dropdown with SuggestionsSelect in landing pages and service overview page {kibana-pull}126679[#126679]
+* Store Logs UI settings in a dedicated `infrastructure-monitoring-log-view` saved object {kibana-pull}125014[#125014]
+
+Platform::
+* The saved objects management table has a new "Last updated" column to easily access recently modified saved objects {kibana-pull}132525[#132525]
+* Sync panels tooltips on dashboard level {kibana-pull}130449[#130449]
+
+Querying & Filtering::
+* Support fields custom label on filter editor {kibana-pull}130533[#130533]
+* Allows comma delimiter on the filters multiple selections dropdowns {kibana-pull}130266[#130266]
+
+Security::
+* Disallow creating a role with the existing name in the role management page. Introduce a new optional boolean `createOnly` parameter in the create role API to prevent overwriting existing roles {kibana-pull}132218[#132218]
```

Comment:
```suggestion
* Disallows creating a role with an existing name in the role management page. Introduces an optional boolean `createOnly` parameter in the create role API to prevent overwriting existing roles; the default value is false, preserving the original API behavior {kibana-pull}132218[#132218]
```